lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSlide.js

```diff
@@ -13,7 +14,30 @@ class CampaignSlide extends React.Component {
     this.controller = new CampaignController();
   }
 
+  componentDidMount() {
+    // get campaigns, this.controller.getCampaigns({
```

Comment:
yeah, was just confusing since it ends in `({`... looks like you commented out part and missed commenting out the closing `});`...